analysis/wwz/wwz4l.py

```diff
@@ -359,12 +359,14 @@ def process(self, events):
 
         # Set up the list of systematics that are handled via event weight variations
         wgt_correction_syst_lst = [
```

Comment:
Can you rename this as `wgt_correction_syst_lst_common`? I think it will make the following lines more clear (since then in the following lines it will be like `wgt_correction_syst_lst = wgt_correction_syst_lst_common + other stuff`)